CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Master (Unreleased)
 
-## 2.21.0 (2023-XX-XX)
+## 2.21.0 (2023-05-03)
 
 - Extracted from `rubocop-rspec` into a separate repository for easier use with Minitest/Cucumber. ([@ydah])
 
```

Comment:
Below is pretty old, we should refer to a more recent ref like 6b1ea4c3.

Reply:
Being old means we need to cherry-pick some commits? Or am I missing something?

Reply:
The link a few lines below that suggest checking earlier changelog in rubocop-r spec points to a version when we extracted rubocop-capybara. We should point to a more recent rubocop-rspec changelog revision that would include changes for 2.20.0, as it has some changes to factory_bot cops.

Reply:
Ah! I see😅 I had completely missed that. Thank you ❤️